### PR TITLE
Generate XML test report

### PR DESCRIPTION
**Pull request by aweng98**

### CODE REVIEW

File: project/Settings.scala [hunk 1]
```diff
@@ -13,6 +13,9 @@ import sbtrelease.ReleasePlugin
 
 object Settings {
 
+  // `-u` - generate xml junit test report
+  val testSettings = List("-u", "test-reports")
```
Comment (danbills) [inline project/Settings.scala]: doesn't look like this is used anywhere?

**Comment by aweng98** [comment]
@danbills Hi Dan, could you please review this PR? Thank you.

**Comment by gemmalam** [comment]
@aweng98 Miguel and Khalid will review this

**Comment by danbills** [comment]
what does this do? @aweng98

**Comment by danbills** [comment]
oh I see this is now scalatest specific.  I think earlier this title had JUnit mentioned.  LGTM :+1:

**Pull request merged**